conda/libmesh/build.sh

```diff
@@ -67,7 +67,8 @@ EOF`
                      --with-vtk-lib=${BUILD_PREFIX}/libmesh-vtk/lib \
                      --with-vtk-include=${BUILD_PREFIX}/libmesh-vtk/include/vtk-${SHORT_VTK_NAME} \
                      --with-methods="opt oprof devel dbg" \
-                     --without-gdb-command
+                     --without-gdb-command \
+                     --with-cxx-std=2014
```

Comment:
Should this be --with-cxx-std-min=2014?  We don't want to force a downgrade on users who might want newer standards in their app codes.

Reply:
Didn't know this was a thing. I agree!